MAUI/ImageEditor/save.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Image save in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about save support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Save in .NET MAUI ImageEditor (SfImageEditor)
+
+The Image Editor control in .NET MAUI allows you to save the edited image as PNG and JPEG.
+
+To save the modified image, you can use the [`Save`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Save_System_Nullable_Syncfusion_Maui_ImageEditor_ImageFileType__System_String_System_String_System_Nullable_Microsoft_Maui_Graphics_Size__) method, which accepts parameters such as file name, file type, file path, and image size. The supported file types for saving are PNG and JPEG. You can also save the image by clicking the save button on the toolbar.
```

Comment:
The supported file types for saving are PNG and JPEG.

Add links for Png and Jpeg. ImageFileType.Png

Reply:
Added a link